umbraco-cloud/product-upgrades/version-specific-upgrades/migrate-from-8-to-latest.md

```diff
@@ -0,0 +1,127 @@
+
+It is not possible to migrate the custom code as the underlying web framework has been updated from ASP.NET to ASP.NET Core. All templates and custom code will need to be re-implemented.
+
+You also need to make sure that the packages that you are using are available on the latest version.
+
+## Prerequisites
+
+* A Umbraco 8 Cloud project running **the latest version of Umbraco 8**.
+* A clean Cloud project running the latest version of Umbraco **at least 2 environments**.
+* A backup of your Umbraco 8 project database.
+
+{% hint style="info" %}
+We strongly recommend having at least **2 environments** on the new Umbraco project.
+
+Should something fail during the migration, the Development environment can be removed and re-added to start over on a clean slate.
+{% endhint %}
+
+## Step 1: Content Migration
+
+1. Create a backup of the database from your Umbraco 8 project using the [database backup guide](../../databases/backups.md).
+    * Alternatively you can clone the environment down and take a backup of the local Database.
+    * Make sure to **restore both content and media** from your Cloud environment.
+2. Import the database backup into SQL Server Management Studio.
+3. Clone down the **Development** environment from the new Cloud project.
+4. Test the project and make sure to log in to the backoffice.
+5. Update the connection string in the new Cloud projects `appsettings.json` file so that it connects to the Umbraco 8 database:
+
+```json
+"ConnectionStrings": {
+    "umbracoDbDSN": "Server=YourLocalSQLServerHere;Database=NameOfYourDatabaseHere;Integrated Security=true"
+}
+```
+
+6. Enable [Unattended Upgrades](https://docs.umbraco.com/umbraco-cms/fundamentals/setup/upgrading#enable-the-unattended-upgrade-feature) to authorize the database upgrade, .
+7. Run the new Cloud project locally.
+8. Wait for the site to finish upgrading.
+9. Stop the site and disable the unattended upgrade.
+10. Run the site and log in using Umbraco ID.
+
+{% hint style="info" %}
+This is **only content migration** and the database will be migrated.
+
+You need to manually upgrade the view files and custom code implementation. For more information, see Step 3 of this guide.
+{% endhint %}
+
+## Step 2: File Migration
+
+1. The following files/folders need to be copied into the new Cloud project:
+    * `~/Views` - **Do not** overwrite the default Macro and Partial View Macro files unless changes have been made to these.
+    * `~/Media`
+    * Any files/folders related to Stylesheets and JavaScript.
+2. Migrate custom configuration from the Umbraco 8 configuration files (`.config`) into the `appsettings.json` file on the new Cloud project.
+    * As of Umbraco 9, configuration no longer live in the `Web.Config` file and has been replaced by the `appsettings.json` file.
+3. [Migrate Umbraco Forms data to the database](../../deployment/umbraco-forms-on-cloud.md), if relevant.
```

Comment:
I think it should link to this article instead: https://our.umbraco.com/Documentation/Add-ons/UmbracoForms/Developer/Forms-in-the-Database/

If you scroll down in the Umbraco Forms on Cloud article, it refers back to the above-mentioned article

Reply:
Giving it some more thought, we might need to migrate/add the Forms article to the docs site so that it doesn't refer to Our site anymore.